### PR TITLE
code examples and interactive examples with fiori

### LINKED ISSUE #1176: Interactive examples style
Move relevant styling into the Yari codebase for interactive examples. Clean up SASS as appropriate.

### LINKED ISSUE #1260: Code examples
Add styling for code example such as below:

Screenshot 2020-09-15 at 10 13 02

### PR DESCRIPTION
**Pull request by schalkneethling**
Add code for code examples and interactive examples.
This also addresses the problems with responsiveness.

fix #1260
fix #1176

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
   "devDependencies": {
     "@babel/core": "^7.11.1",
     "@mdn/dinocons": "^0.0.7",
-    "@mdn/minimalist": "^0.0.17",
+    "@mdn/minimalist": "^0.0.27",
```
Comment (peterbe) [inline package.json:92]: Perhaps we can set up Dependabot to have it make PRs against the `mdn-fiori` branch too. 
Having said that, I think it's healthy to often and periodically merge in `master` to keep the branch up-to-date too.

Comment (schalkneethling) [reply package.json:92]: Yup, I do that after each feature branch has been merged and before I create a new feature branch of `mdn-fiori`. The `minimalist` updates I do manually though as I need to add/change things there.

**Comment by peterbe** [comment]
Fiori
Screen Shot 2020-09-16 at 9 05 13 AM

Kuma
Screen Shot 2020-09-16 at 9 05 42 AM

I see that Kuma has 15px padding. And Fiori only has a 6/12px. 
I'm not an expert at this and it's hard not to be biases based on what you're used to but I quite like fatter padding because it makes it "stand out" a bit more. 
[web.dev](https://web.dev/vitals/) uses 16px.

**Comment by peterbe** [comment]
The interactive examples seem to work. 

And the TOC on mobile seems ok now. 
Screen Shot 2020-09-16 at 9 43 16 AM

**Comment by peterbe** [comment]
One thing I don't quite understand but something I nevertheless miss is; the sidebar doesn't have the "Related Topics" headline.
Screen Shot 2020-09-16 at 9 44 34 AM

Perhaps that's not Fiori related, but Yari related. I just haven't noticed how weird it looks to go straight into the sidebar links without a header.

**Comment by schalkneethling** [comment]
> One thing I don't quite understand but something I nevertheless miss is; the sidebar doesn't have the "Related Topics" headline.

Yup, on my todo list when I tackle the sidebar.

**Comment by schalkneethling** [comment]
> [web.dev](https://web.dev/vitals/) uses 16px.

Let me bump that.

**Comment by schalkneethling** [comment]
> > [web.dev](https://web.dev/vitals/) uses 16px.
> 
> Let me bump that.

Whoop, that happens in minimalist so, will update there.

**Comment by schalkneethling** [comment]
Hey @peterbe, thanks for the review. I am going to merge this one so I can move forward with the next items. If there is anything you meant to mention here, please still add it and I will make sure to address it in a follow-up PR. 🌮

**Comment by peterbe** [comment]
The only thing was if you could consider more padding on the pre tags.

**Comment by schalkneethling** [comment]
> The only thing was if you could consider more padding on the pre tags.

Yup, I did bump that up. Thank you, Peter.

**Pull request merged**